kinetic/distribution.yaml

```diff
@@ -9290,6 +9290,28 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/open_manipulator_msgs.git
       version: kinetic-devel
     status: developed
+  open_manipulator_p:
+    doc:
+      type: git
+      url: https://github.com/ROBOTIS-GIT/open_manipulator_p.git
+      version: kinetic-devel
+    release:
+      packages:
+      - open_manipulator_p
+      - open_manipulator_p_control_gui
+      - open_manipulator_p_controller
+      - open_manipulator_p_description
+      - open_manipulator_p_libs
+      - open_manipulator_p_teleop
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/ROBOTIS-GIT-release/open_manipulator_p-release.git
```

Comment:
Just poking around a bit at the release repository, I don't think bloom was successful.  At least, when I click on many of the branches, they are 404.  Can you take a look and maybe re-bloom here?

Reply:
Hi @clalancette, I checked all the kinetic branches in the https://github.com/ROBOTIS-GIT-release/open_manipulator_p-release.git repository, but they are all available as of now. I'll re-bloom anyway just in case I might have missed something during the process.
Thank you for checking.

Reply:
Seems better now, thanks for looking.